server/covmanager/views.py

```diff
@@ -349,8 +349,7 @@ class CollectionViewSet(mixins.CreateModelMixin,
         CollectionFilterBackend
     ]
 
-class RepositoryViewSet(mixins.CreateModelMixin,
-                        mixins.ListModelMixin,
+class RepositoryViewSet(mixins.ListModelMixin,
```

Comment:
Nice, I guess this allowed creating repositories remotely before your change?

Reply:
Yeah.. It isn't a terrible thing but I didn't think it was intended.